src/js/pyodide.ts

```diff
@@ -287,7 +287,11 @@ export async function loadPyodide(
 
   const pyodide = finalizeBootstrap(API, config);
   // API.runPython works starting here.
-
+  if (!pyodide.version.includes("dev")) {
+    // Currently only used in Node to download packages the first time they are
+    // loaded. But in other cases it's harmless.
+    API.setCdnUrl(`https://pyodide-cdn2.iodide.io/v${pyodide.version}/full/`);
```

Comment:
Isn't `iodide.io` url deprecated BTW?

Reply:
Probably. I still primarily use it...

Reply:
Perhaps let's use `https://cdn.jsdelivr.net` as in our documentation.

Reply:
FYI, we should definitely not use it, it doesn't cache well. I'm thinking we should fully discontinue it soon.